contents/en/general/Guide_to_the_Supermatter_en.md

```diff
@@ -0,0 +1,21 @@
+<h1 align='center'>The Supermatter Engine</h1>
+<ol>
+  <li>Wrench canisters in ports in green squares. Plasma for (2) and Nitrogen for (1), but you also can use Plasma for both</li>
+  <li>Turn on pumps in yellow squares. IMPORTANT: Set them to max, else your TEGs won't be efficient</li>
+  <li>Set up omni-filters in red squares. Set it to filter gas that was wrenched in ports in green squares (1). Turn on pump in red square. Wrench empty canister in port in red square.</li>
+  <li>Head to the computer in cyan square. Search for the inject port.</li>
+  <li>(Optional) Double check everything.</li>
+  <li>Start SM.</li>
+</ol>
+
+Usually you don't want your EER be too high, so use about <b>12-16 emitter shots</b> for Nitrogen setup. It will eventually get lower, so check it sometimes.
+
+TEG power output depends on: differences between temperatures in both loops and amounts of moles in both loops. More moles = more power, so it's good idea to introduce more gases to the loops.
+
+If you can keep your cooling good enough you might even leave emitter on, but I would not recommend that.
+
+<img src="https://cdn.discordapp.com/attachments/534648536264998942/535097086245470219/eris-SM-1.jpg">
+
+<img src="https://cdn.discordapp.com/attachments/534648536264998942/535097091891134488/eris-SM-2.jpg">
+
+<b>Additional Info:</b> You can also wrench N2 or Plasma canister in port next to port in red square, and turn on pumps on the left to start cooling waste gases. This is being used to cool waste gases, and change waste canister less frequently. It's recommended to do that, since it's taking <30 seconds and also prevents gases from interacting with each other (plasma+oxygen+heat=in other words fire) in pipes and canister.
```

Comment:
We can't have gas reaction right in pipes, if i remember right

Reply:
well I think you can have reactions in **unwrenched canisters**, isn't it, or i'm simply made some mistakes?

Reply:
I just tested it and it seemed like you can burn gas inside the pipe. I'm not the one who wrote this guide so I don't know everything about bay/eris atmos perfectly but what the person wrote seems to be true. Unless anyone can give a solid confirmation that what the person said is wrong then I'm just going to leave it as is. When I tested to see if gasses could burn in a pipe I wrenched in an oxygen and a plasma canister then heated the pipe they were in with some heat exchange pipes. The reason I say the burning occurred is because I found CO2 in the pipe. The percentages of each gas settled at 7% plasma, 15% carbon dioxide, and 78% oxygen.